shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -725,23 +725,14 @@ - (void)keyboardWillChangeFrame:(NSNotification*)notification {
   CGFloat scale = [UIScreen mainScreen].scale;
 
   // The keyboard is treated as an inset since we want to effectively reduce the window size by the
-  // keyboard height. We also eliminate any bottom safe-area padding since they keyboard 'consumes'
-  // the home indicator widget.
+  // keyboard height. The Dart side will compute a value accounting for the keyboard-consuming
+  // bottom padding.
```

Comment:
Consider re-wording as: "The framework is responsible for computing a value accounting for keyboard-consuming bottom padding." since, in theory it's possible for another framework to be written on top of the engine.

Reply:
This is done in `dart:ui` though, not the framework.